chukwuemekachm/GraphQL-API
# #23 Implements the get publishers query

#### What does this PR do?
- This PR implements the `getPublishers` query.
#### Description of Task to be completed?
- Update the schema with new types
- Modify the errorFormater
- Create a `getPublishers` resolver
- Add more static types
#### How should this be manually tested?
- Set up the repo locally.
- Create a `getPublishers` query
- Add `search`, `limit`, `orderBy`, and `offset` parameters to sort, paginate, and filter as desired
- Observe the result and behaviours.
#### Any background context you want to provide?
#### What are the relevant Github issues?
- [#23](https://github.com/chukwuemekachm/GraphQL-API/issues/23)
#### Screenshots (if appropriate)

<img width="1440" alt="screenshot 2019-01-25 at 1 52 16 am" src="https://user-images.githubusercontent.com/33798252/51748468-d31f0500-20ac-11e9-946a-95a03115b45a.png">


#### Questions:
- N/A
